dbconnect.py

```diff
@@ -0,0 +1,72 @@
+#TODO Needs more work to make it complete
+
+import psycopg2
+
+
+class DBConnect(object):
+    """
+    Class for connecting to a postgresql database using a with statement
+    """
+
+    def __init__(self, dbhost='', db='', dbuser='', dbpass='', dbport=3306, autocommit=False,
```

Comment:
This get's overridden by what comes out of the .cfgnfo file. So, maybe it should be changed to the default PostgreSQL port, or changed to some None type.